sdk/storage/Azure.Storage.DataMovement.Blobs/api/Azure.Storage.DataMovement.Blobs.net6.0.cs

```diff
@@ -42,6 +42,18 @@ public partial class AppendBlobStorageResourceOptions : Azure.Storage.DataMoveme
         public Azure.Storage.Blobs.Models.AppendBlobRequestConditions DestinationConditions { get { throw null; } set { } }
         public Azure.Storage.Blobs.Models.AppendBlobRequestConditions SourceConditions { get { throw null; } set { } }
     }
+    public partial class AzureBlobStorageResourceProvider
```

Comment:
Maybe thinking of dropping the word Azure and just having `BlobStorageResourceProvider`.

Same with the below with `BlobStorageResources`